composer.json

```diff
@@ -18,7 +18,7 @@
     ],
     "require": {
         "php": ">=5.5.0",
-        "psr/http-message": "~1.0",
+        "psr/http-message": "~1.0 || ^2.0",
```

Comment:
I think you can keep ^ for 1.x also.

Reply:
Fixed at 4067b432595f705c854b9d29c0f4ce34076d3b23.

However, the CI for PHP 7.1 is failing. I think this is not directly related to this change due to the following error.

> ==> Setup Tools
> ✗ composer 403: rate limit exceeded

> Composer 2.3.0 dropped support for PHP <7.2.5 and you are running 7.1.33, please upgrade PHP or use Composer 2.2 LTS via "composer self-update --2.2". Aborting.